packages/tocco-ui/src/Panel/Panel.js

```diff
@@ -1,8 +1,11 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 
+import StyledPanel from './StyledPanel'
+
 /**
- * <Panel/> is used to conceal and display related content alternating by interaction.
+ * <Panel/> is used to conceal and display related content alternating by interaction and to
+ * emphasize close relationship of content.
  */
 class Panel extends React.Component {
   state = {
```

Comment:
is this needed? I think a React.Component class has a empty state by default

Reply:
state = {
    isOpen: this.props.isOpen
  }

is needed because initial state is defined by a props.

Alternatively I could rewrite row 37 to `isOpen: this.state.isOpen || this.props.isOpen,`